entries/p/penfed.org.json

```diff
@@ -0,0 +1,8 @@
+{
+  "PenFed": {
+    "categories": "banking",
+    "documentation": "https://www.penfed.org/website-security",
```

Comment:
```suggestion
```
I can't find any documentation on how to set up passkey usage on the page you linked to. Could you please provide another documentation page? 
If no documentation is available or you can't find it, please attach screenshots of the 2FA setup and/or login process.

> [!WARNING] 
> Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)

Reply:
I can't find better docs describing it, but there is a screenshot on [its page in the Apple app store](https://apps.apple.com/us/app/penfed-mobile/id460008350):
<img width="460" height="999" alt="image" src="https://github.com/user-attachments/assets/0765ac93-4766-4bf1-9dc3-e72ae178ef4e" />

Is that sufficient?